Amplify/Core/Error/CoreError.swift

```diff
@@ -10,6 +10,8 @@ public enum CoreError {
 
     /// A related operation performed on `List` resulted in an error.
     case listOperation(ErrorDescription, RecoverySuggestion, Error? = nil)
+    
+    case operation(ErrorDescription, RecoverySuggestion, Error? = nil)
```

Comment:
This is a breaking change.

Reply:
Will remove this, see https://github.com/aws-amplify/amplify-swift/pull/2583#issuecomment-1372540315 for more details

Reply:
removed